tensorrt_llm/_torch/modules/fused_moe/fused_moe_trtllm_gen.py

```diff
@@ -148,7 +151,8 @@ def create_weights(self):
         self._check_configs()
 
         # TODO: FIX this.
```

Comment:
Not strictly related to this PR but looks like this TODO can be removed? It's not really clear what it even refers to

Reply:
I have no idea, either. I can remove it.